app/frontend/src/features/profile/view/leaveReference/formSteps/submit/ReferenceOverview.tsx

```diff
@@ -0,0 +1,77 @@
+import { Card, CardContent, Hidden, Link, Typography } from "@material-ui/core";
+import SliderLabel from "components/RatingsSlider/SliderLabel";
+import TextBody from "components/TextBody";
+import UserSummary from "components/UserSummary";
+import {
+  CONTACT_LINK,
+  CONTACT_US,
+  COUCHER_WAS_APPROPRIATE,
+  COUCHER_WAS_NOT_APPROPRIATE,
+  FURTHER,
+  PRIVATE_REFERENCE,
+  PUBLIC_REFERENCE,
+  RATING,
+  REFERENCE_MOBILE_USER,
+  REFERENCE_SUBMIT_HEADING,
+  THANK_YOU,
+} from "features/profile/constants";
+import { useReferenceStyles } from "features/profile/view/leaveReference/ReferenceForm";
+import React from "react";
+
+import { SubmitReferenceProps } from "./SubmitReference";
+
+export default function ReferenceOverview({
+  user,
+  referenceData,
+}: SubmitReferenceProps) {
+  const classes = useReferenceStyles();
+
+  return (
+    <>
+      <Typography variant="h2">{REFERENCE_SUBMIT_HEADING}</Typography>
+      <TextBody className={classes.text}>{THANK_YOU}</TextBody>
+      <Hidden mdUp>
+        <TextBody className={classes.text}>{REFERENCE_MOBILE_USER}</TextBody>
+        <UserSummary user={user} />
+      </Hidden>
+      <Typography variant="h3" className={classes.text}>
+        {PUBLIC_REFERENCE}
+      </Typography>
+      <Card className={classes.card}>
+        <CardContent>
+          <TextBody className={classes.text}>{referenceData.text}</TextBody>
+        </CardContent>
+      </Card>
+      <Typography variant="h3" className={classes.text}>
+        {PRIVATE_REFERENCE}
+      </Typography>
+      <ul>
+        {referenceData.wasAppropriate === "true" ? (
+          <li>
+            <TextBody className={classes.text}>
+              {COUCHER_WAS_APPROPRIATE}
+            </TextBody>
+          </li>
+        ) : (
+          <li>
+            <TextBody className={classes.text}>
+              {COUCHER_WAS_NOT_APPROPRIATE}
+            </TextBody>
+          </li>
+        )}
```

Comment:
```suggestion
          <li>
            <TextBody className={classes.text}>
              {referenceData.wasAppropriate === "true" ? COUCHER_WAS_APPROPRIATE : COUCHERS_WAS_NOT_APPROPRIATE}
            </TextBody>
          </li>
```
Might want to Prettier format this if you are taking the suggestion directly though, but that'd reduce the repetition